go.mod

```diff
@@ -77,3 +77,5 @@ replace k8s.io/client-go => k8s.io/client-go v0.18.3
 
 // >v1.2.0 has some conflict with prometheus/alertmanager. Hence prevent the upgrade till it's fixed.
 replace github.com/satori/go.uuid => github.com/satori/go.uuid v1.2.0
+
+replace github.com/gocql/gocql => github.com/grafana/gocql v0.0.0-20200604163347-a8642710dbb1
```

Comment:
Could you add a comment to explain why? Will be easier to remember the rationale over the time.